web/js/notifications/wv.notifications.ui.js

```diff
@@ -185,9 +191,9 @@ wv.notifications.ui = wv.notifications.ui || function(models, config) {
 
     type = obj.notification_type;
     idString = obj.created_at.toString();
-    fieldExists = wv.util.browser.localStorage?!!localStorage.getItem(type):false;
+    fieldExists = false;
     fieldValueMatches = false;
-
+    fieldExists = localStorage.getItem(type);
```

Comment:
You need to check `wv.util.browser.localStorage` here before you use `localStorage`. That can either be with a ternary operator, like this;
`wv.util.browser.localStorage?!!localStorage.getItem(type):false;`
Or with two lines, like this;
```
fieldExists = false;
if (wv.util.browser.localStorage) fieldExists = !!localStorage.getItem(type);
```
I prefer the ternary operator, since it's shorter.